### DISCUSSION TITLE
StripeIntegration module for Magento2 Integration with Breeze

### DISCUSSION THREAD

**Discussion by maharzan86** [breezefront/community#255] [3. Integrations]
Hi there,

**Preconditions**
Here is environment I am using

Magento version - 2.4.6-p4
PHP Version - 8.1
StripeIntegration version - 3.5.13

I have installed StripeIntegration module for Magento2 (https://github.com/stripe/stripe-magento2-releases/commit/50183ea0fb1265e7b71825d984b5526c27619c24) it is working but there is an error in some of the pages.
There are mainly errors in 2 places:
1. minicart page
2. My Account > My Payment Methods page

![stripe_minicart](https://github.com/breezefront/community/assets/45053620/05746955-a03e-4b2e-99ae-dafd10ca74ee)

![stripe_my_account](https://github.com/breezefront/community/assets/45053620/57e74b51-8f95-4bed-bf91-083cdedf7121)

Although, there is an error in minicart page, there is no problem in the order placing but it would be great if there are no errors displaying in those pages.
Is there any way to deal with this?
I have attached the screenshot showing the error details.

Thanks in advance.

**Comment by vovayatsyuk** [comment]
We have a Stripe integration module: https://github.com/breezefront/module-breeze-stripe-payments

It will fix the minicart and product page errors. "My payment methods" page is not integrated yet.

**Reply by maharzan86** [reply]
Oh, I see!
I'll try this for minicart and product page.
Thanks a lot!　👍

**Reply by maharzan86** [reply]
Thanks a lot.
The errors in the minicart and product page has been resolved. 👍
Waiting for the solution regarding to the error displaying in "My payment methods" page soon! 🥺 🙏

**Comment by vovayatsyuk** [comment]
The next version (1.1) will include support for "My payment methods". It will also require the next version of breeze module (2.15).
